tagstudio/src/qt/widgets/preview_panel.py

```diff
@@ -499,31 +497,32 @@ def update_widgets(self):
                     )
 
                     # TODO: Do this somewhere else, this is just here temporarily.
-                    extension = os.path.splitext(filepath)[1][1:].lower()
                     try:
                         image = None
-                        if extension in IMAGE_TYPES:
-                            image = Image.open(filepath)
-                        elif extension in RAW_IMAGE_TYPES:
+                        if filepath.suffix in IMAGE_TYPES:
+                            image = Image.open(str(filepath))
+                        elif filepath.suffix in RAW_IMAGE_TYPES:
                             with rawpy.imread(filepath) as raw:
                                 rgb = raw.postprocess()
                                 image = Image.new(
                                     "L", (rgb.shape[1], rgb.shape[0]), color="black"
                                 )
-                        elif extension in VIDEO_TYPES:
+                        elif filepath.suffix in VIDEO_TYPES:
                             video = cv2.VideoCapture(filepath)
```

Comment:
```suggestion
                            video = cv2.VideoCapture(str(filepath))
```
cv2.VideoCapture expects a string here, and throws an error when given a Path object such as:
```
[PreviewPanel][ERROR] Couldn't Render thumbnail for F:\Example 4\Folder A\yoink\video.mp4 (because of OpenCV(4.9.0) :-1: error: (-5:Bad argument) in function 'VideoCapture'
> Overload resolution failed:
>  - Can't convert object to 'str' for 'filename'
>  - VideoCapture() missing required argument 'apiPreference' (pos 2)
>  - Argument 'index' is required to be an integer
>  - VideoCapture() missing required argument 'apiPreference' (pos 2)
)
```
Can be reproduced by checking the console log after clicking on a video file in the library. I see that the other instances outside of `ts_cli.py` are also using this cast.